### ISSUE TITLE
Many Errors and files not found. (still need help!)

### ISSUE THREAD

**Issue by ghost** [nolirium/aroc#20]
Hello, I have been trying to get rooted on both the Canary (73) and Stable (71) of Chrome OS but the same thing happens on both of them. Lot's of errors. I have a HP Chromebook 14 G5 (snappy). 
Here is what I did and exactly what happened. (what it said in the terminal)
crosh> shell
chronos@localhost:/ sudo /usr/share/vboot/bin/make_dev_ssd.sh --remove_rootfs_verification

  ERROR: YOU ARE TRYING TO MODIFY THE LIVE SYSTEM IMAGE /dev/mmcblk1.

  The system may become unusable after that change, especially when you have
  some auto updates in progress. To make it safer, we suggest you to only
  change the partition you have booted with. To do that, re-execute this command
  as:

    sudo /usr/share/vboot/bin/make_dev_ssd.sh --remove_rootfs_verification --partitions 2

  If you are sure to modify other partition, please invoke the command again and
  explicitly assign only one target partition for each time  (--partitions N )
Since that didn't work I did this.
chronos@localhost: sudo /usr/share/vboot/bin/make_dev_ssd.sh --remove_rootfs_verification --partitions $(( $(rootdev -s | sed -r 's/.*(.)$/\1/') - 1))
make_dev_ssd.sh: INFO: Kernel A: Disabled rootfs verification.
make_dev_ssd.sh: INFO: Backup of Kernel A is stored in: /mnt/stateful_partition/backups/kernel_A_20190107_193909.bin
make_dev_ssd.sh: INFO: Kernel A: Re-signed with developer keys successfully.
make_dev_ssd.sh: INFO: Successfully re-signed 1 of 1 kernel(s)  on device /dev/mmcblk1.
(reboots)
chronos@localhost:/ curl -Ls https://raw.githubusercontent.com/nolirium/aroc/master/01Root.sh | sudo sh (everything goes fine....until the middle to the bottom)
id_Images/Mounted/system/bin/daemonsu-service.sh': No such file or directory
chgrp: cannot access '/usr/local/Android_Images/Mounted/system/bin/daemonsu-service.sh': No such file or directory
chcon: cannot access '/usr/local/Android_Images/Mounted/system/bin/daemonsu-service.sh': No such file or directory
Creating file init.super.rc in Android rootfs
Adding daemonsu service to init.super.rc
Adding 'import /init.super.rc' to existing init.rc
Substituting '|mount rootfs rootfs / remount bind rw' for '|mount rootfs rootfs / remount bind ro' in existing init.rc
A backup of init.rc will be stored as init.rc.old
sed: can't read /../init.rc: No such file or directory
Removing temporary files

Done!

Please check the output of this script for any errors.

Please reboot now, then run script 02SEPatch.sh.
What could be going wrong?

**Comment by ghost** [comment]
Does anyone know? I really want to root my chromebook.

**Comment by xe500linux** [comment]
See my reply to issue #21, might work for you. I see you've already tried changing channels, but the script only works for me directly after a fresh return to stable, and the required powerwash (and same remove_rootfs cmd you used before). You could also try the second cmd I posted there before running the root script, and see if it gives a read-only error

**Comment by ghost** [comment]
@xe500linux What do you mean by follow the prompts? I understand everything after that, but what do you mean by that?

**Comment by xe500linux** [comment]
@AshtonGamerYT I just mean waiting for the release channel to download and update, then it should ask you to reboot (downgrading to stable requires powerwash, upgrading to beta/dev doesn't). Basically just try the remove-rootfs and root scripts on a freshly changed channel, with reboots inbetween of course. Even powerwashing it once afterwards (without changing channels again) will cause it to fail again.

**Comment by ghost** [comment]
OK! Gonna try that now!

**Comment by ghost** [comment]
@xe500linux  Nope, same thing. I am not using the combined script, No errors at the start, but then it starts doing the (missing directory/file) thing again at the end....
